lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -540,57 +540,86 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
   patterns.onOp(
       "Squeeze", 13, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
```

Comment:
update version number to 1. the old impl had overly high since-version numbers.

Reply:
No, it can't be changed to 1 since the version number 1 expects inputs to be in different form. Please refer: https://onnx.ai/onnx/operators/text_diff_Squeeze_1_13.html